operations/node_management/Add_Remove_Replace_NCNs/Add_Remove_Replace_NCNs.md

```diff
@@ -28,6 +28,10 @@ All activities required for site maintenance are complete.
 
 The latest CSM documentation has been installed on the master nodes. See [Check for Latest Documentation](../../../update_product_stream/README.md#check-for-latest-documentation).
 
+Note: With respect to add/remove/replace worker nodes, we need to check if selective worker node personalization of iSCSI
```

Comment:
It's not quite accurate to say that they need to check if it's being used, because there are things that may be necessary even if it's not currently being used (for example, they are adding a new worker NCN, and they do not not want it used for iSCSI). I think it's better to just tell them that before they begin the add or remove worker procedure, they should go read the relevant section on the Managing selective node personalization page.